packages/cli/package.json

```diff
@@ -92,8 +102,6 @@
     "optimize-css-assets-webpack-plugin": "^5.0.1",
     "ora": "^3.0.0",
     "postcss-loader": "^3.0.0",
-    "preact": "^8.1.0",
```

Comment:
Does this break prerendering?

Reply:
it shouldn't, as a peerDep it should resolve fine